src/main/java/frc/robot/constants/Constants2023.java

```diff
@@ -18,6 +18,8 @@ public class Constants2023 extends GameConstants {
     public static final int SHOOTER_MOTOR_ID_1 = 4400; //needs to be changed
     public static final int SHOOTER_MOTOR_ID_2 = 4500; //needs to be changed
 
+    public static final boolean DEPLOYER_DEBUG = true;
```

Comment:
This should go in GameConstants.java. Put it with the rest of the debug constants.